packages/integrations/cloudflare/src/index.ts

```diff
@@ -138,6 +138,7 @@ export default function createIntegration(args?: Options): AstroIntegration {
 						const fileName = entryPointsRouteData[index].component
 							.replace('src/pages/', '')
 							.replace('.astro', '.js')
+							.replace('.ts', '.js')
```

Comment:
This logic is not very robust, because a page in Astro can have many extensions: js, ts, mjs, cjs, md, mdx, markdoc, etc

Maybe we should review the logic

Reply:
That's true, I'll refactor that logic to be more robust

Reply:
@ematipico updated the logic to make it more robust. currently it has to a lot of memory allocations for clear code. let me know if I should consolidate all the steps into one `.map()`?